be/src/column/stream_chunk.h

```diff
@@ -0,0 +1,105 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "column/chunk.h"
+#include "column/chunk_extra_data.h"
+
+namespace starrocks::vectorized {
+
+using Int8ColumnPtr = std::shared_ptr<vectorized::Int8Column>;
+
+enum StreamRowOp : std::int8_t { INSERT = 0, DELETE = 1, UPDATE_BEFORE = 2, UPDATE_AFTER = 3 };
+using StreamRowOps = std::vector<StreamRowOp>;
+
+using StreamChunk = Chunk;
+using StreamChunkPtr = std::shared_ptr<StreamChunk>;
+
+/**
+ * `StreamChunk` is used in Incremental MV which contains a hidden `ops` column, the `ops` column indicates
+ * the row's operation kind, eg: INSERT/DELETE/UPDATE_BEFORE/UPDATE_AFTER.
+ * 
+ * `StreamChunkConverter` is used as a converter between the common `Chunk` and the `StreamChunk`.
+ */
+class StreamChunkConverter {
+public:
+    static StreamChunkPtr make_stream_chunk(ChunkPtr chunk, Int8ColumnPtr ops) {
```

Comment:
explicitly, you already have noticed that Int8Column can used as StreamRowOps,  so we just use FixedLengthColumn, is it ok?